doc/user/content/ingest-data/mysql/self-hosted.md

```diff
@@ -0,0 +1,154 @@
+---
+title: "Ingest data from self-hosted MySQL"
+description: "How to stream data from self-hosted MySQL database to Materialize"
+menu:
+  main:
+    parent: "mysql"
+    name: "Self-hosted"
+    identifier: "mysql-self-hosted"
+---
+
+{{< private-preview />}}
+
+This page shows you how to stream data from a self-hosted MySQL database to
+Materialize using the [MySQL source](/sql/create-source/mysql/).
+
+## Before you begin
+
+{{% mysql-direct/before-you-begin %}}
+
+## Step 1. Enable GTID-based binlog replication
+
+Before creating a source in Materialize, you **must** configure your MySQL
+database for GTID-based binlog replication. This requires the following
+configuration changes:
+
+Configuration parameter          | Value  | Details
+---------------------------------|--------| -------------------------------
+`log_bin`                        | `ON`   |
+`binlog_format`                  | `ROW`  | This configuration is [deprecated as of MySQL 8.0.34](https://dev.mysql.com/doc/refman/8.0/en/replication-options-binary-log.html#sysvar_binlog_format). Newer versions of MySQL default to row-based logging.
+`binlog_row_image`               | `FULL` |
+`gtid_mode`                      | `ON`   | In the AWS console, this parameter appears as `gtid-mode`.
+`enforce_gtid_consistency`       | `ON`   |
+`binlog retention hours`         | 168    |
```

Comment:
underscore separators missing => `binlog_retention_hours`

Reply:
This is the name of the config parameter in AWS ([docs](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/mysql-stored-proc-configuring.html#mysql_rds_set_configuration-usage-notes.binlog-retention-hours)).

Reply:
This means we should probably adjust all parameters to match the service-specific documentation, though.